### ISSUE TITLE
What is the test data for Argoverse

### ISSUE THREAD

**Issue by TsuTikgiau** [kami93/CMU-DATF#3]
Hello,
I notice that we don't have ground truth prediction data in argoverse test set. Do the numbers in paper come from the validation set? Thank you!

**Comment by os1a** [comment]
Yes, I think they are coming from the validation set.
To report on the testing set, you need to submit to the benchmark at:
https://evalai.cloudcv.org/web/challenges/challenge-page/454/overview

However, in the benchmark the number of future trajectories should be 6, while in the paper they use 12.

This of course would open a second question:
The authors mentioned in the supplementary material of their paper that they use early stopping when overfitting is observed. Does it mean that they use the validation set to measure the overfitting and do the early stopping. If so, then how did they report their results on the validation set?

**Comment by kami93** [comment]
Hi, we used the validation sets provided by datasets for the report. For the early stopping, we used a small subset out of the training set for the validation purpose.

In conclusion, it is clear to consider "The validation set" officially provided by datasets as "Testing set" in this model, and you may derive your own "validation set" by splitting "The training set" officially provided by datasets.

**Comment by kami93** [comment]
Hi @TsuTikgiau, yes they are from the validation set as officially provided by datasets.

It is clear to consider "The validation set" officially provided by datasets as "Testing set" in this model, and you may derive your own "validation set" by splitting "The training set" officially provided by datasets.

**Comment by TsuTikgiau** [comment]
Hello @kami93 
thank you for your answer! May I ask about the way you split the training set, like the ratio and do you split them randomly or has a specified order?

**Comment by kami93** [comment]
> Hello @kami93
> thank you for your answer! May I ask about the way you split the training set, like the ratio, and do you split them randomly or has a specified order?

Unfortunately, I cannot find specific parameters for the validation set in my note. As far as I remember, we took 10% subset out of a shuffled data cache. I think this setting will work.

**Comment by TsuTikgiau** [comment]
@kami93 Thanks for your answer!
